src/scss/main.scss

```diff
@@ -10,6 +11,7 @@
 // Layout
 
 @import 'layout/header';
+@import 'layout/team-section';
```

Comment:
the styles for this component should be in the components folder !
Please update this and import from components: @import 'components/team-section';

Reply:
OMG! sorry! 
will do!